base/libgit2/walker.jl

```diff
@@ -1,5 +1,31 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    GitRevWalker(repo::GitRepo)
+
+A `GitRevWalker` *walks* through the *revisions* (i.e. commits) of
+a git repository `repo`. It is a collection of the commits
+in the repository, and supports iteration and calls to [`map`](@ref)
```

Comment:
I can add a docstring for `LibGit2.map`, for some reason (???) I thought we already had one.